### PR TITLE
stream node changes to db

### PR DESCRIPTION
**Pull request by RoiGlinik**

### CODE REVIEW

File: src/robusta/core/discovery/discovery.py [hunk 1]
```diff
@@ -83,6 +84,17 @@ def __create_service_info(
             is_helm_release=is_helm_release,
         )
 
+    @staticmethod
+    def create_service_info(obj: Union[Deployment, DaemonSet, StatefulSet, Pod, ReplicaSet]) -> ServiceInfo:
```
Comment (ganeshrvel) [inline src/robusta/core/discovery/discovery.py:88]: any particular reason we need this wrapper method? cant we just use the exisiting `__create_service_info`?

**Review by ganeshrvel — CHANGES_REQUESTED** [review]
Good work. Left some comments.

File: playbooks/robusta_playbooks/event_enrichments.py [hunk 2]
```diff
@@ -251,40 +255,14 @@ def external_video_enricher(event: ExecutionBaseEvent, params: VideoEnricherPara
 @action
 def resource_events_diff(event: KubernetesAnyChangeEvent):
     new_resource = event.obj
+    if not isinstance(new_resource, (Deployment, DaemonSet, StatefulSet, Node)):
+        return
+    elif isinstance(new_resource, Pod) and (new_resource.metadata.owner_references or is_pod_finished(new_resource)):
```
[the remark this answers is not in the document]
Comment (RoiGlinik) [reply playbooks/robusta_playbooks/event_enrichments.py:260]: yea you are right, ill update it

**Review by ganeshrvel — APPROVED** [review]

File: src/robusta/core/sinks/robusta/robusta_sink.py [hunk 3]
```diff
@@ -524,3 +532,32 @@ def __publish_new_namespaces(self, namespaces: List[NamespaceInfo]):
...
+
+            if cache.resource_version >= int(new_node.metadata.resourceVersion or 0):
+                return
+
+            new_info = self.__from_api_server_node(new_node, [])
+            new_info.memory_allocated = cache.memory_allocated
+            new_info.cpu_allocated = cache.cpu_allocated
+            new_info.pods_count = cache.pods_count
+            new_info.pods = cache.pods
+
```
Comment (RoiGlinik) [inline src/robusta/core/sinks/robusta/robusta_sink.py:555]: check if equal and pass

**Pull request merged**